tests/Fixtures/app/config/config_mysql.yml

```diff
@@ -0,0 +1,10 @@
+imports:
+    - { resource: config_test.yml }
+
+doctrine:
```

Comment:
It should use environment variables to allow to run test locally (same for Postgres).